.gitignore

```diff
@@ -15,3 +15,5 @@
 
 # Ignore Byebug command history file.
 .byebug_history
+
+.DS_Store
```

Comment:
You'll need to `git rm --cached .DS_Store` to tell git to no longer track the file, and then the `.gitignore` will take effect